packages/web-app/src/axiosFactory.tsx

```diff
@@ -48,7 +48,16 @@ export const createClient = (): AxiosInstance => {
       try {
         if (error.response.status === 401 && error.config.baseURL === config.apiBaseUrl) {
           const store = getStore()
-          store.auth.setIsAuthenticated(false)
+          if (error.response.headers.get('www-authenticate') === 'Sudo-Cookies') {
+            store.profile.setPendingProtectedAction({
+              method: error.response.config.method,
+              url: error.response.config.url,
+              body: error.config.data,
```

Comment:
let's use the same naming as used in `error` 

```suggestion
              data: error.config.data,
```

Reply:
But it's actually a request body
`data` name won't be clear for use in other methods

Reply:
in `axios` codebase it's called `data`, let`s follow the same naming for consistency

Reply:
Changed it